### PR TITLE
Patch 2

### LINKED ISSUE #23: Ammo
just configs not setup i guess for ammo in weapons that you have added,tried the protector 9mm and the pm 9mm. no ammo is available

### LINKED ISSUE #24: gang hideout 3
no access to get inside building after capturing, i guess someone bought it?

### LINKED ISSUE #36: Service Station - Needs updating
https://github.com/AsYetUntitled/Framework/blob/master/Altis_Life.Altis/core/fn_setupStationService.sqf#L29-L63

Who's excited to do this? :D

### LINKED ISSUE #39: Escape Range (Tanoa Jail)
As per the title, needs raising

### LINKED ISSUE #40: RPT Spam 'REMOTE has no unit'
https://github.com/AsYetUntitled/Framework/blob/master/Altis_Life.Altis/core/functions/fn_playerTags.sqf#L60

### LINKED ISSUE #41: Wanted System Cleanup
Need's a bloody good cleanup

### LINKED ISSUE #46: Rental car && Garage spawn
Hi, 
When i try to put a rental car in garage, i have a hint : " Try to store your vehicle ..." but why he don't tell me " Sorry you can't store because it's a rental car ! " ?? ;) 

Edit : When a vehicle is on the spawn point and i want to span another on the same point, the hint tell me : Spawning vehicle please wait..

### PR DESCRIPTION
**Pull request by BoGuu**
Needs a good test!

This will: Close #46, close #41, close #40, close #39, close #36, close #24, close #23.

### CODE REVIEW

**Comment by Jawshy** [comment]
Anything in particular or everything related to the wanted system?

**Comment by BoGuu** [comment]
@Jawshy 

There is no change except the tidiness and performance of the code. So if it all works as it did before, jobs a good'un :+1:

File: Altis_Life.Altis/core/pmenu/fn_wantedInfo.sqf [hunk 1]
```diff
@@ -5,28 +5,27 @@
...
 */
-private["_display","_list","_crimes","_bounty","_mylist","_data"];
 disableSerialization;
 
-_data = [_this,0,[],[[]]] call BIS_fnc_param;
-_display = findDisplay 2400;
-_list = _display displayCtrl 2402;
-_mylist = [];
-_data = call compile format["%1", _data];
+private _data = param [0,[],[[]]];
```
Comment (tkcjesse) [inline Altis_Life.Altis/core/pmenu/fn_wantedInfo.sqf]: @BoGuu You said we were doing params always :/

Comment (BoGuu) [inline Altis_Life.Altis/core/pmenu/fn_wantedInfo.sqf]: Old, fear not

**Comment by Jawshy** [comment]
Closed as https://github.com/AsYetUntitled/Framework/pull/79 was merged.

**Pull request closed without merging**